.github/workflows/docker.yml

```diff
@@ -1,19 +1,19 @@
 name: Build Base Docker Image
 
-# Triggered only if environment.yml in main changes
+# Triggered only if environment_meson.yml in main changes
```

Comment:
Has the corresponding Dockerfile  and Docker image creation been updated? I cannot see any indication of this 

I can push some changes is you do not mind

Reply:
Please do, I don't know how to do that unfortunately

Reply:
We can unify these two now if that's easier. Meson is becoming the default, so no need to avoid installing all the extra bits it needs in the default `environment.yml`.

Reply:
I checked in with Melissa - and agreed to leave it for now (it is more a quick fix to have this ready for Mentored Sprints tomorrow)

I can then go back and clean this as it makes the whole build messier